assets/component-cart-items.css

```diff
@@ -290,6 +302,10 @@ cart-remove-button .icon-remove {
     padding-top: 4rem;
   }
 
+  .cart-items .cart-item__nested-line td {
+    padding-top: 1rem;
```

Comment:
This also affects the drawer's second line with the quantity selector. Are you sure it should be that way?